front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -28,6 +40,18 @@ const StdcmViewV2 = () => {
           pathProperties={pathProperties}
           setPathProperties={setPathProperties}
           setCurrentSimulationInputs={setCurrentSimulationInputs}
+          setStdcmSimulationResults={setStdcmSimulationResults}
+          setSimulationCreationDate={setSimulationCreationDate}
+        />
+      )}
+      {trainScheduleV2Activated && rollingStock && stdcmSimulationResults && (
```

Comment:
You can replace your test by `stdcmV2Results?.stdcmResponse`
```suggestion
      {trainScheduleV2Activated && rollingStock && stdcmV2Results?.stdcmResponse && (
```

Reply:
fix